api/tests/unit/infrastructure/serializers/jsonapi/certification-point-of-contact-serializer_test.js

```diff
@@ -1,50 +1,100 @@
-const { expect, domainBuilder } = require('../../../../test-helper');
+const { expect, domainBuilder, sinon } = require('../../../../test-helper');
 const certificationPointOfContactSerializer = require('../../../../../lib/infrastructure/serializers/jsonapi/certification-point-of-contact-serializer');
 
 describe('Unit | Serializer | JSONAPI | certification-point-of-contact-serializer', function() {
 
   describe('#serialize()', function() {
+    let clock;
 
-    it('should convert a CertificationReferent model object into JSON API data', function() {
+    afterEach(function() {
+      clock.restore();
+    });
+
+    it('should convert a CertificationPointOfContact model into JSON API data', function() {
       // given
-      const certificationCenter = { id: 1, name: 'Serre tiff', type: 'SCO', externalId: 'externalId', isRelatedOrganizationManagingStudents: false };
-      const certificationPointOfContact = domainBuilder.buildCertificationPointOfContact({ certificationCenters: [certificationCenter] });
+      clock = sinon.useFakeTimers(new Date('2021-09-01'));
```

Comment:
Ce test devrait casser quand on changera les dates dans AllowedCertificationCenterAccess. Il faudrait expliciter l'origine de la date:
```suggestion
      const ACCESSIBLE_COLLEGE_DATE = new Date('2021-09-01')
      clock = sinon.useFakeTimers(ACCESSIBLE_COLLEGE_DATE);
```